### PR TITLE
Alh/fetch foo properties

### PR DESCRIPTION
**Pull request by wolfsage**
- Support fetchOtherFoos even if fetchFoos isn't specified
 - Support fetchFooProperties / fetchOtherFooProperties

### CODE REVIEW

File: lib/Ix/DBIC/ResultSet.pm [hunk 1]
```diff
@@ -1075,7 +1093,15 @@ sub ix_get_list_updates ($self, $ctx, $arg = {}) {
 
     unless ($is_removed) {
       FILTER: for my $filter (keys $arg->{filter}->%*) {
-        if (differ($entity->$filter, $arg->{filter}{$filter})) {
+        my $diff;
+
+        if (my $differ = $filter_map->{$filter}->{differ}) {
```
Comment (rjbs) [inline lib/Ix/DBIC/ResultSet.pm:1130]: I don't see any tests for custom differs.

**Comment by rjbs** [comment]
If I read all this correctly, and I think I did, it look quite good!  :+1:

**Comment by wolfsage** [comment]
Right, need to add some here and to topicbox.

Also, just pushed a5c8f34 which is fairly important (until we validate arguments more strictly)

**Comment by rjbs** [comment]
Thanks.  Yes, I believe this is all improvements.

**Pull request merged**